docs/setup.rst

```diff
@@ -140,16 +144,17 @@ should have a running Re:dash instance with everything you need to get
 started . Re:dash should be available using the server IP or DNS name
 you assigned to it. You can point your browser to this address, and login
 with the user "admin" (password: "admin"). But to make it useful, there are
-a few more steps that you need to manually do to complete the setup:
+a few manual steps to complete the setup:
 
-First ssh to your instance and change directory to ``/opt/redash``. If
-you're using the GCE image, switch to root (``sudo su``).
+First connect to your instance using ssh, and switch to root (``sudo su``).
+
+If you're using the Docker image, ``docker exec -i -t redash_redash_1 /bin/bash``, will put you in ``/opt/redash/current`` as root.
 
 Users & Google Authentication setup
 -----------------------------------
 
-Most of the settings you need to edit are in the ``/opt/redash/.env``
-file.
+Most of the settings you need to edit are in the ``/opt/redash/current/.env``
+file. If it doesn't exist, ``cp /opt/redash/current/.env.example /opt/redash/current/.env``.
```

Comment:
The instructions assume you either use one of the provided images or the bootstrap script. The `.env` File will always be there and `/opt/redash/current/.env` is a symlink to `/opt/redash/.env` (so it's not lost when upgrading).

Note that for Docker you should *not* use the .env file but rather Docker's env variables.